e2e/testsuite/grpc_query.go

```diff
@@ -132,6 +153,17 @@ func (s *E2ETestSuite) QueryClientStatus(ctx context.Context, chain ibc.Chain, c
 	return res.Status, nil
 }
 
+// QueryCurrentPlan queries the currently scheduled plans, if any
```

Comment:
```suggestion
// QueryCurrentPlan queries the currently scheduled plans.
```

 The original wording makes it sound like there is special behaviour in the case of no plans being found.